korman/exporter/physics.py

```diff
@@ -77,6 +77,14 @@ def generate_physical(self, bo, so, bounds, name=None):
             physical.sceneNode = self._mgr.get_scene_node(bl=bo)
 
             getattr(self, "_export_{}".format(bounds))(bo, physical)
+            
+            if self._exporter().has_coordiface(bo):
+                if not physical.mass:
+                    # ...Since the object has a coordinate interface but isn't a kickable, it's likely going to be animated. In such case,
+                    # we must warn Plasma to have the collisions follow the object's coordinates as well.
+                    physical.mass = 1.
+                    simIface.setProperty(plSimulationInterface.kPinned, True)
+                    physical.setProperty(plSimulationInterface.kPinned, True)
```

Comment:
Oh, wait, I see what you're saying based on the forum post. Have you tried *only* setting this mass? I'm a little wary of setting random flags just because it works. kPinned does nothing in CWE.